locales/pl/_excludes.json

```diff
@@ -75,5 +75,6 @@
     "Uzbekistan",
     "Vanuatu",
     "Zambia",
-    "Zimbabwe"
+    "Zimbabwe",
+    "OK",
```

Comment:
It was simple - file was broken))

![image](https://github.com/user-attachments/assets/37bd6fe2-7539-45c4-87d6-c48951e17c6d)

Reply:
sorry! fat fingers